eclipse-collections-forkjoin/pom.xml

```diff
@@ -63,7 +63,21 @@
         <plugins>
 
             <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>${jacoco.plugin.version}</version>
```

Comment:
@Godin I tried to fix it such that redundancy is avoided, and everything from the parent pom is used. But JaCoCo execution agent was not getting triggered without adding the artifact and version. May be I was missing something basic. If you get a chance can you please help to remove the redundancy? I can close this PR if you would like to raise one with JaCoCo integrated correctly. 
It is ok if you cannot, I will try to fix it, and push a new commit. 
Appreciate your review!